src/models/MatrixAction.ts

```diff
@@ -187,9 +187,13 @@ export class MatrixAction {
                 }
 
                 if (filename) {
-                    url += `/${filename}`;
+                    url += `/${encodeURIComponent(filename)}`;
+                    text = `${fileSize} < ${url} >`;
```

Comment:
If it's not an empty string, `fileSize` starts with a space.
Both lines that use `fileSize` in your change seem to expect `fileSize` to not have an additional space.

Reply:
This will still start with a space.
I don't think that's what you expect.